src/main/java/ch/njol/skript/lang/function/FunctionReference.java

```diff
@@ -27,6 +27,9 @@
  */
 public class FunctionReference<T> implements Contract, Executable<Event, T[]> {
 
+	static final String AMBIGUOUS_ERROR = "Skript cannot determine which function named '%s' to call. " +
```

Comment:
Do you think it's possible to get more specific errors? For example, consider these functions:
```
function a(text: text, number: number)
function a(number: number, number: number)
```
If I call `a({_var}, 10)`, could we instead have an error like: `Skript cannot determine which function named 'a' to call as the first parameter could be a/an <possible types list>. Try clarifying the type of the argument using syntax like '<first possible type> within {_a}'`.